doc/getting-started/getting-started.rst

```diff
@@ -328,11 +328,11 @@ Generate a Board Configuration File
          disk="/media/$USER/"$(ls /media/$USER)
          cp -r "$disk"/board_inspector ~/acrn-work
 
-#. On the target, load the ``msr`` driver, used by the Board Inspector:
+#. Before using Board Inspector, you need to update the pciid information:
```

Comment:
Strictly speaking, I don't think we need to, but it's a good idea and a good suggestion. How about the following text (looking at @dbkinder to help finalize it ;-)

"On the target, update the ``pci.ids`` file which contains the list of known identifiers related to PCI devices"

Reply:
How about this:
```
#. On the target, you can optionally update the ``pci.ids`` file
   that contains the list of known identifiers for PCI devices:
```